mobile/lib/modules/asset_viewer/ui/remote_photo_view.dart

```diff
@@ -140,6 +138,62 @@ class _RemotePhotoViewState extends State<RemotePhotoView> {
       );
     }
 
+    if (widget.loadOriginal) {
```

Comment:
How come we duplicate this whole block of code, don't we already have these condition in this function?

Reply:
Maybe cleaning up this code block would reveal why the original asset isn't requested.

Reply:
I figured it out. I was messing up with my `cacheKey` by asking for the `getThumbnailCacheKey`. Just sloppiness. It's fixed now.

Reply:
I think I removed any use of this widget, but I'll clean this block up or revert my changes if necessary.